code/modules/cm_marines/orbital_cannon.dm

```diff
@@ -419,24 +449,32 @@ var/list/ob_type_fuel_requirements
 	var/datum/cause_data/cause_data = create_cause_data(initial(name), source_mob)
 	cell_explosion(target, clear_power, clear_falloff, EXPLOSION_FALLOFF_SHAPE_LINEAR, null, cause_data) //break shit around
 	sleep(clear_delay)
```

Comment:
It seems a little weird this explosion doesn't cause a shake until after the sleep.

Reply:
the code for the main explosion is right under and it only executes if the turf is valid. This is a much smaller initial explosion and I feel like this shouldn't warrant the camera shake unless the main larger explosion executes.

Reply:
I suppose, but cell_explosion causes a sfx, and all other instances of cell_explosion here are with a ob_shake. So its weird this is the only one that has a sound, then delay, then there's a shake from a different explosion.

Just experiment with this. If you're happy with it as is, just let me know here and its resolved.

Reply:
I suppose, but cell_explosion causes a sfx, and all other instances of cell_explosion here are with a ob_shake. So its weird this is the only one that has a sound, then delay, then there's a shake from a different explosion.

Just experiment with this. If you're happy with it as is, just let me know here and its resolved.

Reply:
I'm happy with it.